packages/gestalt/src/SegmentedControl.js

```diff
@@ -36,6 +36,7 @@ export default function SegmentedControl(props: Props) {
             key={i}
             onClick={e => onChange({ event: e, activeIndex: i })}
             role="tab"
+            type="submit"
```

Comment:
@pfarejowicz could we change this to `type="button"`?

Reply:
I can make the change, I agree `button` is better than `submit`. My goal was to introduce the minimal amount of changes and since this is a `<button />` and not a `<Button />` the default here was actually always setting this to `type="submit"` since we were not specifying it explicitly. See the [Mozilla docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button) for reference. Making this change now, but just making a not here for future readers that the behavior is changing from our released version.

> **type**
> The type of the button. Possible values are:
>
> **submit**: The button submits the form data to the server. This is the default if the attribute is not specified, or if the attribute is dynamically changed to an empty or invalid value.